mdpow/tests/test_ensemble.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import absolute_import
+
+from . import tempdir as td
+
+import os.path
+
+import py.path
+
+import pybol
+
+import numpy as np
+
+from ..analysis.ensemble import Ensemble, EnsembleAnalysis, EnsembleAtomGroup
+from gromacs.utilities import in_dir
+
+from pkg_resources import resource_filename
+RESOURCES = py.path.local(resource_filename(__name__, 'testing_resources'))
+MANIFEST = RESOURCES.join("manifest.yml")
+
+ensemble_keys = [('water', 'Coulomb', '0000'),
```

Comment:
add a comment that we want to add tests for octanol later

Reply:
done